.github/workflows/test.yml

```
@@ -14,34 +12,10 @@ jobs:
-    - name: Install PocketIC server
-      uses: dfinity/pocketic@main
-      with: 
-        pocket-ic-server-version: "9.0.3"
-
-    - name: Confirm PocketIC installation
-      run: ${POCKET_IC_BIN} --version
-
-    - name: Build
-      run: cargo build --release 
+        nix_path: nixpkgs=channel:nixos-unstable
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)